fixtures/missing-translations/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "missing-translations"
```

Comment:
When running the `bin/cli` script from a fixture, the `let rootDir = pkgDir.sync();` line would return the root of this whole repo, because there is no `package.json` in fixtures. Adding it with just a name fixed this.